e2e/tests/121-Portal/QuickFilter/QuickFilterBulkActionsBug.spec.ts

```diff
@@ -0,0 +1,55 @@
+import { SeedScript } from '@121-service/src/scripts/seed-script.enum';
+import {
+  bulkUpdateRegistrationsCSV,
+  importRegistrationsCSV,
+} from '@121-service/test/helpers/registration.helper';
+import {
+  getAccessToken,
+  resetDB,
+} from '@121-service/test/helpers/utility.helper';
+import { expect, test } from '@playwright/test';
+import HomePage from '../../../pages/Home/HomePage';
+import LoginPage from '../../../pages/Login/LoginPage';
+import TableModule from '../../../pages/Table/TableModule';
+
```

Comment:
```suggestion
import NLRCProgram from '@121-service/seed-data/program/program-nlrc-ocw.json';
```

Reply:
It's better to prevent hardcoded program names which you can get from the json 
and than you can later change this
`await homePage.openPAsForRegistrationOcwProgram('NLRC OCW program');`
to 
`NLRCProgram.titlePortal.en`